examples/example_surface_profiling.py

```diff
@@ -0,0 +1,48 @@
+import slam.surface_profiling as surfpf
```

Comment:
please format your example following e.g. example_curvature.py
This will show you the format that is imposed by the doc tools to create the online documentation

Reply:
ok